s3-other.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2020 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "aws_s3_bucket" "apiary_logs_bucket" {
+  count  = var.apiary_log_bucket == "" ? 1 : 0
```

Comment:
`apiary_logs_bucket` vs `apiary_log_bucket` is really confusing, especially since enabling one disables the other.  Can we use a more descriptive name?  Maybe something like `apiary_managed_logs_bucket` (or similar), and add to the CHANGELOG that it is only enabled if the customer does not supply their own logs bucket with the variable `apiary_log_bucket`?